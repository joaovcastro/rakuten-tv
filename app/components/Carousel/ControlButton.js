import styled from 'styled-components';

const ControlButton = styled.div`
  width: 5%;
  display: flex;
  height: 400px;
  justify-content: center;
  align-items: center;
  justify-content: center;
  cursor: pointer;
`;

export default ControlButton;
