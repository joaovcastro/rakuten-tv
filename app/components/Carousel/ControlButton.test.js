import React from 'react';
import { shallow } from 'enzyme';

import ControlButton from './ControlButton';

describe('<ControlButton />', () => {
  it('correctly renders', () => {
    const wrapper = shallow(<ControlButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
