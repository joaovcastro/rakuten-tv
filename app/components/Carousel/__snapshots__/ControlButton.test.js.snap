// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ControlButton /> correctly renders 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  width: 5%;
  display: flex;
  height: 400px;
  justify-content: center;
  align-items: center;
  justify-content: center;
  cursor: pointer;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
